HACKING.md

```diff
@@ -1,17 +1,28 @@
 # Hacking on snapd
 
-Hacking on snapd is fun and straightforward. The code is extensively
-unit tested and we use the [spread](https://github.com/snapcore/spread)
-integration test framework for the integration/system level tests.
+Hacking on `snapd` is fun and straightforward. The code is extensively unit
+tested and we use the [spread](https://github.com/snapcore/spread) integration test framework for the
```

Comment:
Please keep the line lenght as it was, I think this document might be using 79 chars as the limit.